.devcontainer/Dockerfile

```diff
@@ -53,9 +41,3 @@ RUN git clone https://github.com/zsh-users/zsh-syntax-highlighting ${ZSH_CUSTOM:
 
 # Copy our custom .zshrc to user's home directory
 COPY .zshrc /home/node/.zshrc
-
```

Comment:
The below workaround is replaced by the "postStartCommand" in devcontainer.json (next file)